src/server/src/main/java/io/cassandrareaper/ReaperApplicationConfiguration.java

```diff
@@ -508,7 +508,20 @@ public String toString() {
     /* We require jmx access to all nodes in the local datacenter */
     LOCAL,
     /* Each datacenter requires at minimum one reaper instance that has jmx access to all nodes in that datacenter */
-    EACH
+    EACH;
+
+    /**
+     * Check if the current datacenter availability mode is to have collocation between Reaper and a DC/node.
+     * @return true if we're in a collocated mode, false otherwise
+     */
+    public boolean isInCollocatedMode() {
+      switch (this) {
+        case EACH:
+          return true;
+        default:
+          return false;
+      }
+    }
```

Comment:
i suspect this doesn't need to be part of this PR? (including the related patch chunk in `SegmentRunners.handlePotentialStuckRepairs(..)`)

Reply:
Reverted this change to make it part of the sidecar PR instead.